lib/Net/Fastly/Client.pm

```diff
@@ -3,6 +3,7 @@ package Net::Fastly::Client;
 use strict;
 use warnings;
 use JSON::XS;
+use Mozilla::CA;
```

Comment:
Maybe make this a soft requirement by delaying the evaluation of `SSL_DEFAULTS` options?